exercises/practice/yacht/src/test/scala/YachtTest.scala

```diff
@@ -0,0 +1,120 @@
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class YachtTests extends AnyFunSuite with Matchers {
+
+  test("Yacht") {
+    Yacht.score(Category.Yacht, List(Die.Five, Die.Five, Die.Five, Die.Five, Die.Five)) shouldEqual 50
+  }
+
+  test("Not Yacht") {
+    Yacht.score(Category.Yacht, List(Die.One, Die.Three, Die.Three, Die.Two, Die.Five)) shouldEqual 0
```

Comment:
All but the first test should be `pending`:

```suggestion
    pending
    Yacht.score(Category.Yacht, List(Die.One, Die.Three, Die.Three, Die.Two, Die.Five)) shouldEqual 0
```

Could you also update the other tests?

Reply:
Added `pending` for the tests.